cmd/goal/bundledGenesisInject.go

```diff
@@ -0,0 +1,2793 @@
+// Code generated by bundle_genesis_json.sh, along with langspec.json. DO NOT EDIT.
```

Comment:
so close but so far: https://pkg.go.dev/embed

Reply:
it is only for 1.16+ folks, not for us